hack/lib/golang.sh

```diff
@@ -97,15 +99,20 @@ readonly KUBE_TEST_PORTABLE=(
   hack/lib
 )
 
-# If we update this we need to also update the set of golang compilers we build
-# in 'build/build-image/Dockerfile'
+# If we update this we should also update the set of golang compilers we build
+# in 'build/build-image/cross/Dockerfile'
+# Right now, onsi/ginkgo isn't compiling on linux/arm64, so it's outcommented
+# That should be fixed in the future. However, as a server platform it works fine
 readonly KUBE_CLIENT_PLATFORMS=(
   linux/amd64
   linux/386
   linux/arm
+#  linux/arm64
+  linux/ppc64le
   darwin/amd64
   darwin/386
   windows/amd64
+  windows/386
```

Comment:
Because I saw that a compiler in the earlier `kube-build:cross` was built for `windows/386`, but it wasn't listed as a client platform. Supposed it just had been forgotten.

But it doesn't harm to build `kubectl` for 32-bit win also.